pkg/bgpv1/manager/route_policy_reconciler_test.go

```diff
@@ -31,7 +31,7 @@ var (
 			},
 		},
 		Spec: v2alpha1api.CiliumLoadBalancerIPPoolSpec{
-			Cidrs: []v2alpha1api.CiliumLoadBalancerIPPoolCIDRBlock{
+			Cidrs: []v2alpha1api.CiliumLoadBalancerIPPoolIPBlock{
```

Comment:
This test is part of the BGP package, I am honestly not sure why tests from that package are using LB-IPAM in these tests instead of just manually setting `.status.LoadBalancer.ingress[]`. I don't think its necessary to add extensive coverage of LB-IPAM features here.